product_docs/docs/epas/15/database_administration/02_edb_loader/data_loading_methods.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Overview of data loading methods"
```

Comment:
This should be the parent topic and Conventional path loading and direct path loading should be the child topics under this one